File: src/main/scala/com/bombproj/vo/ProjectAllFileVO.java
package com.bombproj.vo;

import lombok.Data;

@Data
public class ProjectAllFileVO {

    private String id;
    private String fileName;
    private String suffix;
    private Long size;
    private String creator;
    private String creatorName;
    private String updateTime;
    private Integer type;
    private Integer starred;
    private Integer shared;
    private String version;
    private String parentId;
    private String fullPath;

}

File: src/main/scala/com/bombproj/vo/ProjectAllFileVO.scala
package com.bombproj.vo

import scala.beans.BeanProperty

case class ProjectAllFileVO() {

    @BeanProperty var id: String = _
    @BeanProperty var fileName: String = _
    @BeanProperty var suffix: String = _
    @BeanProperty var size: Long = _
    @BeanProperty var creator: String = _
    @BeanProperty var creatorName: String = _
    @BeanProperty var updateTime: String = _
    @BeanProperty var `type`: Int = _
    @BeanProperty var starred: Int = _
    @BeanProperty var shared: Int = _
    @BeanProperty var version: String = _
    @BeanProperty var parentId: String = _
    @BeanProperty var fullPath: String = _

}
